recipes/sqlalchemy-oso/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "sqlalchemy-oso" %}
+{% set build_number = 0 %}
+{% set version = "0.26.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/osohq/oso/archive/refs/tags/sqlalchemy-v{{ version }}.tar.gz
+  sha256: 121d1b420c17250d57253d84004dcb7c8087579436c86a60f701a139e4781064
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install languages/python/sqlalchemy-oso -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - cffi >=1.0.0
+  run:
+    - python >=3.6
+    - packaging ~=20.9
```

Comment:
don't use the `~` operator: different python build tools have it map to different things, and you don't awlayas get the same thing from conda.